Jobsway2go.WebUI/Models/Section.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+[DataContract]
+public class Section
+{
+    [DataMember]
```

Comment:
Remove also `using System.ComponentModel.DataAnnotations;`, `[DataContract]` and `[DataMember]`